pyoverkiz/const.py

```diff
@@ -83,4 +83,10 @@
         manufacturer="Somfy",
         configuration_url=None,
     ),
+    "ubiwizz": OverkizServer(
```

Comment:
You found the url within the apk?

Reply:
Yes! I will verify this with the user that report it on HA community forum, before merging.

Reply:
Confirmed working!